src/ert/_c_wrappers/job_queue/external_ert_script.py

```diff
@@ -23,13 +23,15 @@ def run(self, *args) -> None:
         # The job will complete before stdout and stderr is returned
         stdoutdata, stderrdata = self.__job.communicate()
 
-        self._stdoutdata = codecs.decode(stdoutdata, "utf8", "replace")
-        self._stderrdata = codecs.decode(stderrdata, "utf8", "replace")
+        stdoutdata = codecs.decode(stdoutdata, "utf8", "replace")
+        stderrdata = codecs.decode(stderrdata, "utf8", "replace")
 
-        sys.stdout.write(self._stdoutdata)
+        sys.stdout.write(stderrdata)
 
         if self.__job.returncode != 0:
-            raise Exception(self._stderrdata)
+            raise Exception(stderrdata)
```

Comment:
Raising a general exception here seems not a good idea, probably want a custom error contected to a handler somewhere or a `RuntimeError`.